CONTRIBUTING.md

```diff
@@ -30,13 +30,13 @@ sbt shell.
 ```sh
 # Fast unit tests for rules, cli, core. Contains a lot
 # of different test suites, so it's recommended to use testOnly.
-> unit2_13Target2_13/test
+> unit2_13/test
```

Comment:
@rvacaru once you rebase your PR against this, you'll have to use `unit3` instead of `unit3Target3`